<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveChangesDialog</class>
 <widget class="QDialog" name="SaveChangesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Changes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 63, 65);
color: rgb(230, 230, 230);


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>The following files have unsaved changes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
	height: 30px;
	width: 60;
}
QPushButton:hover {
	background-color: rgb(60, 187, 255);
}</string>
         </property>
         <property name="text">
          <string>Save All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="doNotSaveButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
	height: 30px;
	width: 80;
}
QPushButton:hover {
	background-color: rgb(60, 187, 255);
}</string>
         </property>
         <property name="text">
          <string>Do not Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
	height: 30px;
	width: 60;
}
QPushButton:hover {
	background-color: rgb(60, 187, 255);
}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SaveChangesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
